adage/adage/config.py.template

```diff
@@ -136,9 +141,30 @@ CI_CONFIG.update({
                         'PORT': '5434',  # v. 9.4
                      },
     'data':          {
-                        'data_dir': DATA_CONFIG['data_dir'],
-                        'annotation_file': DATA_CONFIG['annotation_file'],
+                        'data_dir': '/home/rof/clone/data/',
```

Comment:
Any chance we can use the DATA_CONFIG here? This would allow it to still use this configuration for CircleCI.